format/spec.md

```diff
@@ -631,6 +632,29 @@ When expiring snapshots, retention policies in table and snapshot references are
     2. The snapshot is not one of the first `min-snapshots-to-keep` in the branch (including the branch's referenced snapshot)
 5. Expire any snapshot not in the set of snapshots to retain.
 
+#### Statistics file
+
+Statistics files are valid [Puffin files](../puffin-spec). Statistics are informational. A reader can choose to
+ignore statistics information. Statistics support is not required to read the table correctly.
+
+Statistics files' metadata within `statistics` table snapshot field is a struct with the following fields:
+
+| v2         | Field name                      | Type                                   | Description                                                                                            |
+|------------|---------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------------------|
+| _required_ | **`statistics-path`**           | `string`                               | Path of the statistics file. See [Puffin file format](../puffin-spec).                                 |
+| _required_ | **`file-size-in-bytes`**        | `long`                                 | Size of the statistics file.                                                                           |
+| _required_ | **`file-footer-size-in-bytes`** | `long`                                 | Size of the statistics file's footer. See [Puffin file format](../puffin-spec) for footer definition.  |
+| _required_ | **`source-sequence-number`**    | `long`                                 | Table sequence number at which the stats were calculated                                               |
```

Comment:
Shouldn't this also include a snapshot ID at which the stats were calculated?

Reply:
> Also, if this is to be used in v1,

my intention was for this to be v2 feature.

> Shouldn't this also include a snapshot ID at which the stats were calculated?

also, or instead of?

please advise

Reply:
Yes, I think this should have the snapshot for at which stats were calculated.

There's also no reason to not support this in v1. As long as the snapshot ID is here for tracking, it should work just fine.

Reply:
Changed to `source-snapshot-id` and made the whole thing supported in v1.

Reply:
A side question -- now that we have v2 and we will have v3 of the spec, why do we want to add a new feature to v1?